sketch.js

```diff
@@ -1,158 +1,91 @@
-/////////////////////////////////////////////////////
-//                                                 //
-//     Neural Network Training with the Titanic    //
-//                                                 //
-/////////////////////////////////////////////////////
-
-// By: Lydia Jessup
-// Date: August 21, 2019
-// Description: Using TF.js to train an neural network to predict survival on the titanic
-// Also:  This is heavily based off of examples by Dan Shiffman and the tf.js team
-
-
-
-///////////////////////////////////////////////
-// Import, normalize and transform data
-///////////////////////////////////////////////
-
-// this is a standard procedure to normalize values between 0 and 1
-function normalize(value, min, max) {
-  if (min === undefined || max === undefined) {
-    return value;
-  }
-  return (value - min) / (max - min);
+let $ageInput,
+  $ageInputText,
+  $fareInput,
+  $fareInputText,
+  $sexInput,
+  $predictBtn,
+  $predictionText,
+  $status;
+let sample = [0, 0, 0];
+
+
+const TRAIN_DATA_PATH = 'titanic_train.csv';
+const TEST_DATA_PATH = 'titanic_test.csv';
+
+let nn;
+
+let data;
+const neuralNetOptions = {
+  inputs: 3,
+  outputs: 2,
+  debug: true,
+  outputLabels: ['survived'],
+  inputLabels: ['age', 'fare', 'is_female'],
```

Comment:
this is really great having the labels specified this way!

Reply:
Oh, actually I'm confused. We probably shouldn't use the term "labels"  as we're using this term for "class" elsewhere in the library. This is really the "field name" or "data heading", hmmmmm. And these are really part of the data and not the neural network itself. . .leading me again towards something like `ml5.data()`, hmmmm. For now maybe just:

```javascript
  outputs: ['survived'],
  inputs: ['age', 'fare', 'is_female']
```

Reply:
This also allows users to load up data w/ extraneous fields.